CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Contributing to the TinyMCE developer documentation
+
+This page outlines how you can provide feedback and contribute to TinyMCE documentation.
+
+## Contributor Code of Conduct
+
+For our Contributor Code of Conduct, visit our [Contributor Covenant Code of Conduct on GitHub](https://github.com/tinymce/tinymce-docs/blob/develop/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
+
+## How to contribute to TinyMCE's documentation
+
+**TL;DR** A quick way to improve the docs is to click the "Was this article helpful?" and select "no". Then if you want to help more click the "opening an issue" link and submit a GitHub issue.
+
+## So, how can you help?
+
+We welcome people with technical writing experience to help make TinyMCE's docs as useful to our community as possible. Although you don't need to be an expert developer to contribute, a basic understanding of software development is helpful.
+
+What is more important is your ability to articulate complex ideas. Our goal is to help developers understand TinyMCE's functionality as quickly as possible.
+
+Simply branch the [docs GitHub repository](https://github.com/tinymce/tinymce-docs) and submit a PR, or reach out to the [TinyMCE docs team](https://github.com/tinymce/tinymce-docs-4x/issues/new?assignees=&labels=question&template=question.md) if you have a specific question.
+
+## TinyMCE documentation tech stack
+
+The documentation is built with Jekyll. Text files are written in [Markdown](https://help.github.com/articles/markdown-basics/) and published on [GitHub](https://github.com/tinymce/tinymce-docs). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
+
+## Markdown & code syntax
+
+The documentation is written in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/). If you are not familiar with Markdown [read this](https://help.github.com/articles/markdown-basics/).
```

Comment:
This isn't correct to my knowledge. It's using "kramdown" flavored markdown instead: https://kramdown.gettalong.org/